templates/includes/delete_form.html

```diff
@@ -0,0 +1,5 @@
+<form action="" method="post">
+  {% csrf_token %}
+  <input class="btn btn-danger" type="submit" value="Yes, delete!">
+</form>
+  
```

Comment:
add new empty line

Reply:
only I didn't understand why it was needed